src/tpl/CMakeLists.txt

```diff
@@ -135,7 +135,7 @@ if (NOT TARGET camp)
   endif()
 endif ()
 
-if (NOT TARGET fmt::fmt)
+if (NOT TARGET ${UMPIRE_FMT_TARGET})
   if (DEFINED fmt_DIR)
     find_package(fmt REQUIRED
       NO_DEFAULT_PATH
```

Comment:
You missed a few uses of fmt::fmt a couple lines down (calls to set_target_properties and blt_convert_to_system_includes).